components/src/widget/widget.ts

```diff
@@ -59,19 +72,11 @@ export class PaymentWidget extends LitElement {
       return
     }
 
-    let walletAddressInfo: WalletAddress
     try {
-      const walletAddressUrl = checkHrefFormat(
-        toWalletAddressUrl(walletAddress)
-      )
-
-      walletAddressInfo = await getWalletAddress(walletAddressUrl)
+      const walletAddressInfo = await this.validateWalletAddress(walletAddress)
```

Comment:
```suggestion
const response = await fetch(
  `${API_URL}/wallet?walletAddress=${encodeURIComponent(walletAddress)}`
)

const walletAddressInfo = await response.json()
if (!response.ok) {
  throw new Error(
    walletAddressInfo.error.message
  )
}

this.configController.updateState({
  walletAddress: walletAddressInfo as WalletAddress
})
```

by using, for example, `https://ilp.interledger-test.dev/kjmitchelljr` as a wallet address for an address that does not exist, the widget should display the message “This wallet address does not exist.” returned by the api.
Also, I think it's more cleaner if called directly without creating `validateWalletAddress`